src/api/config.py

```diff
@@ -94,23 +99,120 @@ def _check_config() -> None:
         raise ValueError(error_messages.CONFIG_SIGFIGS_FALLBACK_VALID_RANGE)
 
 
+def _set_default_config() -> None:
+    global configuration  # pylint: disable=global-statement
+
+    configuration = _default_config
```

Comment:
I'm not sure why you define a function for the one-line statement

```py
configuration = _default_config
```

that you could just place like that at the end of the file?